rest/plugin.spec.yaml

```diff
@@ -71,6 +71,13 @@ connection:
     type: credential_username_password
     required: false
     example: '{"username": "user@example.com", "password": "mypassword"}'
+  fail_on_error:
+    title: Fail on Error
+    description: Indicates whether the plugin should fail on standard HTTP errors
+    type: boolean
+    required: false
+    default: true
```

Comment:
Let's keep the previous behavior so plugin upgrades are easiser- I think it was the equivalent of false but if not then let us know.

Reply:
Currently the plugin is returning a PluginException if there is an HTTP error, I set the default value to true, to keep the previous behavior 🙂

Reply:
Thanks!